pipeline/tests/asm.rs

```diff
@@ -127,6 +129,8 @@ fn empty() {
 }
 
 #[test]
+// TODO: https://github.com/powdr-labs/powdr/issues/2292
```

Comment:
I don't think this is really worth fixing, see my comment in #2292. But in principle, we could track which namespace identities are generated in.